docs/products/kafka/index.rst

```diff
@@ -0,0 +1,48 @@
+Aiven for Apache Kafka
+======================
+
+What is Aiven for Apache Kafka?
+-------------------------------
+
+Aiven for Apache Kafka is a fully managed **distributed data streaming platform**, deployable in the cloud of your choice. Apache Kafka is an open source data streaming platform, ideal for event-driven applications, near-real-time data transfer and pipelines, stream analytics, and many more applications where a lot of data needs to move between applications in a speedy manner.
+
+Kafka stores a potentially large number of small data payloads, usually for a limited period of time. The storage is organised into "topics" and "partitions" so that many data streams can be handled at once, regardless of how many producers or consumers you need to work with.
```

Comment:
"how many producers or consumers you need to work with." what are producers and consumers?
Maybe we can say 
"regardless of how many applications will produce or consume data."?